presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMinMaxByAggregation.java

```diff
@@ -84,7 +82,8 @@ public void testAllRegistered()
 
     private static List<Type> getTypes()
     {
-        List<Type> simpleTypes = METADATA.getTypeManager().getTypes();
+        List<Type> simpleTypes = new ArrayList<>(METADATA.getTypeManager().getTypes());
+        simpleTypes.removeAll(ImmutableList.of(CODE_POINTS, JSON_PATH, JONI_REGEXP, RE2J_REGEXP, LIKE_PATTERN));
```

Comment:
We discussed this and decided a better way is to check that `getJavaType()` returns a supported type.